cmd/mirror.go

```diff
@@ -221,80 +118,192 @@ func newMirrorSetCmd() *cobra.Command {
 	return cmd
 }
 
+// the `mirror introduce` sub command
+func newMirrorIntroduceCmd() *cobra.Command {
+	name := ""
+	privPath := ""
+
+	cmd := &cobra.Command{
+		Use:   "introduce <id>",
+		Short: "introduce a new owner",
+		Long:  "introduce a new owner to current mirror",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) < 1 {
+				return cmd.Help()
+			}
+
+			id := args[0]
+			if name == "" {
+				name = id
```

Comment:
We assigned a default value for name, do we need to print some message to hint the user? e.g:
```go
fmt.Printf("The --name hasn't specified value, default to %s\n", id).
```

Reply:
OK